CASMTRIAGE-6355 Update command (#4512)
Cray-HPE/docs-csm: operations/configuration_management/Management_Node_Personalization.md

    [[ -z ${FAILED} ]] && echo "No errors" || echo "ERROR: There were errors clearing the CFS state for the following nodes:${FAILED}"
    ```

> > **NOTE:** Do replace `node` with proper value for the cluster being used. The different options of `--type` parameter is obtained by using command `cray hsm state components list --help`.

Reviewer: ```suggestion
> **NOTE:** Replace `<node>` with the value for the cluster being used. The different options for the `--type` parameter can be viewed with the `cray hsm state components list --help` command.
```
